pycti/entities/opencti_case_incident.py

```diff
@@ -242,49 +238,47 @@ def __init__(self, opencti):
         """
 
     @staticmethod
-    def generate_id(name, case_type, created):
+    def generate_id(name):
         name = name.lower().strip()
-        case_type = case_type.lower().strip()
-        if isinstance(created, datetime.datetime):
-            created = created.isoformat()
-        data = {"name": name, "case_type": case_type, "created": created}
+        data = {"name": name}
         data = canonicalize(data, utf8=False)
         id = str(uuid.uuid5(uuid.UUID("00abedb4-aa42-466c-9c01-fed23315a9b7"), data))
-        return "case--" + id
+        return "case-incident--" + id
 
     """
-        List Cases objects
-
+        List Case Incident objects
+        
         :param filters: the filters to apply
         :param search: the search keyword
         :param first: return the first n rows from the after ID (or the beginning if not set)
         :param after: ID of the first row for pagination
-        :return List of Case objects
+        :return List of Case Incident objects
     """
 
     def list(self, **kwargs):
         filters = kwargs.get("filters", None)
         search = kwargs.get("search", None)
-        first = kwargs.get("first", 100)
+        first = kwargs.get("first", 500)
```

Comment:
What was the reason to change from 100 to 500 ?

Reply:
No reason at all, that's actually doesn't really matter